Set wind_pluggin to zero for Gazebo Plane model

Antiheavy opened the pull request:
As documented here https://github.com/PX4/sitl_gazebo/issues/287 the Gazebo wind_pluggin doesn't work properly for the Gazebo Plane model.  This PR sets the wind_pluggin values to zero for the Plane model so that it will fly correctly in simulation.

In the long term, we should figure out exactly why the wind_pluggin does not work properly with gazebo plane model.  I suspect it might have something to do with improper or incomplete modeling of the airplane aerodynamics as it relates to the wind_pluggin, but I don't have the coding skills to really figure this out for sure. 

Questions:
1) if this PR is viewed as too much a hack, feel free to reject it, or suggest improvements.
2) maybe there are other vehicle models that are also affected?  Should we fix it for them too?

Antiheavy committed:
Set wind_pluggin to zero for Gazebo Plane model

As documented here https://github.com/PX4/sitl_gazebo/issues/287 the Gazebo wind_pluggin doesn't work properly for the Gazebo Plane model.  This PR sets the wind_pluggin values to zero for the Plane model so that it will fly correctly in simulation.  In the long term, we should figure out exactly why the wind_pluggin does not work properly with gazebo plane model.  I suspect it might have something to do with improper or incomplete modeling of the airplane aerodynamics as it relates to the wind_pluggin, but I don't have the coding skills to really figure this out for sure. 
Questions:
1) if this PR is viewed as too much a hack, feel free to reject it, or suggest improvements.
2) maybe there are other vehicle models that are also affected?  Should we fix it for them too?

TSC21 reviewed (changes requested):
> > Do we want to disable the plugin entirely?
> 
> That is another option. I suggested setting the values to zero in the plane models for this PR because it is the method I've used and tested in SITL; so I know it works.

I think it's preferable to disable the wind plugin entirely by comment it and also add a comment above it explaining why it is deactivated.

TSC21 commented on models/plane/plane.sdf:
```diff
@@ -404,9 +404,9 @@
-      <xyzOffset>1 0 0</xyzOffset>
-      <windDirection>0 1 0</windDirection>
-      <windForceMean>0.7</windForceMean>
+      <xyzOffset>0 0 0</xyzOffset>
```
@Antiheavy as simple as:
```suggestion
    <!-- Wind plugin deactivated as the wind plugin is not affecting the control surfaces lift and drag.
         This requires an update on the lift/drag plugin as well. -->
    <!-- <plugin name='wind_plugin' filename='libgazebo_wind_plugin.so'>
      <frameId>base_link</frameId>
      <linkName>base_link</linkName>
      <robotNamespace/>
      <xyzOffset>1 0 0</xyzOffset>
      <windDirection>0 1 0</windDirection>
      <windForceMean>0.7</windForceMean>
      <windGustDirection>0 0 0</windGustDirection>
      <windGustDuration>0</windGustDuration>
      <windGustStart>0</windGustStart>
      <windGustForceMean>0</windGustForceMean>
    </plugin> -->
```

Antiheavy committed:
Update models/plane/plane.sdf

commited suggestion from TSC21 to comment out the Wind Plugin for fixed wing model.

Co-Authored-By: Nuno Marques <n.marques21@hotmail.com>

Antiheavy committed:
reverted original changes

removed lines that set the wind plugin values to zero.  Now the original wind plugin values are there, but the whole wind plugin is deactivated with a comment explaining why.
